scripts/setup-agent.ps1

```diff
@@ -178,6 +178,35 @@ function Install-Snort {
     }
 }
 
+# Step 4: Download and install Wazuh Agent Status with error handling
+function Install-AgentStatus {
+    try {
+        Write-Host "Downloading and executing Wazuh Agent Status installation script..."
+
+        $AgentStatusUrl = "https://raw.githubusercontent.com/ADORSYS-GIS/wazuh-agent-status/refs/heads/chore/issue-6/scripts/install.ps1"  # Update the URL if needed
```

Comment:
Please change this to the url of the main branch, since it is were we put our stable release.

- Please create a PR for this branch https://github.com/ADORSYS-GIS/wazuh-agent-status/tree/chore/issue-6 
- merge it to the main 
- change this URL to point to the script of the main branch